google-cloud-spanner/src/test/java/com/google/cloud/spanner/ChannelUsageTest.java

```diff
@@ -220,6 +221,9 @@ private SpannerOptions createSpannerOptions() {
   @Test
   public void testCreatesNumChannels() {
     try (Spanner spanner = createSpannerOptions().getService()) {
+      assumeFalse(
+          "GRPC-GCP is currently not supported with multiplexed sessions",
+          isMultiplexedSessionsEnabled(spanner));
```

Comment:
This test works for the normal channel pool, it is only with Gcp pool that it fails.

```suggestion
          enableGcpPool && isMultiplexedSessionsEnabled(spanner));
```

Reply:
I had originally done this. But it still introduced a bit of flaky behavior. I will be following up to re-enabled this test pending some investigation.

Reply:
Are we using a random channel hint? If so, then I understand why it is flaky. The test iterates 2*numChannels and then checks that all channels have been used. That will often, but not always, be the case if we use random channel affiliation. The test will probably become acceptably non-flaky if we increase the number of iterations to something like 50*numChannels.